src/scss/config/color/_ccs-defaults.scss

```diff
@@ -3,10 +3,9 @@
 /// The Cascading Color System has a few settings
 /// that we can most easily override with CSS variables
 /// @colors ccs-colors
-/// @type map
 /// @group config-color
-$ccs-colors: (
-  'ccs-neutral--theme': var(--ccs-h--prime),
-  'ccs-background': var(--bg),
-  'ccs-color': var(--text),
-);
+
+/* stylelint-disable-next-line scss/dollar-variable-pattern */
+$ccs-neutral--theme: var(--ccs-h--prime);
```

Comment:
We might want to update our `scss/dollar-variable-pattern` setting to allow this pattern, or else do a find/replace to remove the double-dashes?

Reply:
@jgerigmeyer do you have a preference? I would need a few days to decipher this line (@jerivas here's one example of when that regex lesson would come in handy): 
`^([_|-]*[a-z][a-z0-9]*)(-[a-z0-9]+)*-*$`

Reply:
Playing around in RegExr a bit, we could allow:

- 1 or 2 dashes in the middle of a name: `^([_|-]*[a-z][a-z0-9]*)(-{1,2}[a-z0-9]+)*-*$`
- any number of dashes in the middle: `^([_|-]*[a-z][a-z0-9]*)(-*[a-z0-9]+)*-*$`

Currently we're allowing any number at the start and end of the name.

Reply:
Oh nice! TIL about RegExr. I was previously using https://regex101.com/r/vx05jF/1 but the explainer on RegExr is much better ✨

Reply:
You can save any regex and get a link BTW regexr.com/6ri58